README.rst

```diff
@@ -263,17 +265,17 @@ Launch development server and SMTP deubgging server:
 
 .. code-block:: shell
 
-    ./manage.py runserver
+    python3 manage.py runserver
     # open another session and run
-    python -m smtpd -n -c DebuggingServer localhost:1025
+    python3 -m smtpd -n -c DebuggingServer localhost:1025
 
 You can access the admin interface of the test project at http://127.0.0.1:8000/admin/.
 
 Run tests with:
 
 .. code-block:: shell
 
-    ./runtests.py
+    python3 runtests.py
```

Comment:
If you will use virtualenv then you dont need to replace it :)

Reply:
I know that. But still thanks.

Reply:
don't do this please

Reply:
Ok, but just for reference purposes, why?

Reply:
Maybe he too should just introduce for virtualenv ?

Reply:
Yeah, I was thinking about that